### ISSUE TITLE
Categories for groups that generate pages with consolidated events feeds/aggregate impact

### ISSUE THREAD

**Issue by janetgunter** [TheRestartProject/Fixometer#233]
We've already talked about Admins being able to create categories for groups, in relation to process/stage of group development, for the purposes of exporting data for dashboard or something like that.

What would also be useful is for us to be able to create categories for universities (Refer), secondary schools, countries (England, for Nesta), geographic areas (London). So in the backend, we would categorise groups. 

Ideally we would have the option publishing the aggregate impact and events feeds for these groups to the website. These pages could be super important, becoming landing pages for schools and universities. But they could also become national (localised) landing pages. 

@JamesPickstone @ugomatic would love your feedback.

**Comment by JamesPickstone** [comment]
Agreed with all that. The easier we can make it to slice data according to our own reporting priorities (as well as those of individual groups) the better.

I've already had a conversation about this with a group that has fairly complex reporting requirements. They run two types of event across a couple of different areas. They'd like the ability to group their data by type of group and area when reporting back to funders:

> E.g. Reports for:
>  _Area A_ and _Groups 1 and 2_
> _Areas A & B_ and _Group 1_ etc.

I wonder whether a lot of this could be accomplished through tags. Perhaps we could have a set of internal tags (e.g. Nesta) and allow local groups the ability to create their own tags for their events (e.g. venue/area, event type etc.)?

Tags could then (in theory) be used to filter data. It would be super helpful to be able to set up custom reports based on one or multiple tag(s)

**Comment by ugomatic** [comment]
Agreed. Tags could probably also help filter data by type of event (groups directly affiliated as Restarters vs others involved in other ways). A couple of extra suggestions:
1) One extra feature request is displaying on external websites the aggregated impact stats for a "group of groups" matching some specific tags. This could come very handy for the the upcoming Refer website, and could build from the existing iframe embedding we have for individual groups (and events). 
2) The iframe we have implemented focuses on impact stats. Building on Janet's idea of a personalised "event feed", could we have a calendar feature embeddable in other websites too? This could increase the uptake of announcing events on the Fixometer (rather than creating an event only after it took place to upload repair data)

**Comment by ngm** [comment]
I'll just try to summarise these (and why they would be useful):

* ability to categorise groups and events (let's say by using tags) - to facilitate the following features..
* ability to filter stats by those tags (including combinations of tags) - to encourage groups' use of the Fixometer, by allowing for complex reporting when reporting back to funders
* ability to create public landing pages (including the data vis of the stats) based on these tags - to encourage groups' use of the Fixometer 
* ability to filter the embedded stats that orgs can publish on their own sites - to encourage groups' use of the Fixometer and to help with Refer project
* ability to publish event feeds and calendars based on these tags - to encourage groups' use of the Fixometer (this ties to Stephan's request for extra event management features)

Would be good to think out some concrete examples of what e.g.:
* a school's landing page would include (how similar / different to existing group pages)
* a national landing page would include (ditto)
* what a complex report might include (e.g. for the group mentioned by @JamesPickstone )
* what would be useful to be embedded on the Refer website
